<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressBarWidget</class>
 <widget class="QWidget" name="ProgressBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProgressBarWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextBrowser" name="txtLog"/>
   </item>
   <item>
    <widget class="QProgressBar" name="proBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>终止</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFinish">
       <property name="text">
        <string>完成</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
